packages/rich-text/src/create.js

```diff
@@ -104,6 +105,26 @@ function toFormat( { tagName, attributes } ) {
 	};
 }
 
+const RichTextInternalData = Symbol( 'RichTextInternalData' );
+
+export class RichTextData {
```

Comment:
I know it's still early, but it would be great to have at least some documentation on this class and why it exists, even if it's only a sentence. Especially against `RichTextValue` it's confusing to find both and not have anything indicating which is appropriate under which conditions